utils/mp-server/query-scripts/run-queries.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+for sql in $(find . -name '*.sql')
+do
+    mysql < $sql > ${sql}.tsv
```

Comment:
Missing quotes on the $sql expansions.

Reply:
If you wanted to remove the .sql part you can use: `"${sql%.sql}.tsv"`

Reply:
Fixed.  And the output filename having the ".sql" in it or not isn't really important.